cmake/modules/FindGMock.cmake

```diff
@@ -174,31 +184,35 @@ else()
         LOG_CONFIGURE ON
         LOG_BUILD ON
         CMAKE_ARGS ${GTEST_CMAKE_ARGS}
+        BUILD_BYPRODUCTS "${GTEST_LIBRARY}"
+        BUILD_BYPRODUCTS "${GTEST_MAIN_LIBRARY}"
+        BUILD_BYPRODUCTS "${GMOCK_LIBRARY}"
+        BUILD_BYPRODUCTS "${GMOCK_MAIN_LIBRARY}"
     )
 endif()
 
 add_library(libgmock IMPORTED STATIC GLOBAL)
 add_dependencies(libgmock gmock)
+add_library(libgtest IMPORTED STATIC GLOBAL)
+add_dependencies(libgtest gmock)
```

Comment:
I'd suggest that you don't use `add_dependencies` at all. If the `BUILD_BYPRODUCTS` _and_ `IMPORTED_LOCATION` are set correctly then the dependencies should be resolved by CMake itself without declaring them explicitly.